web/src/store/metamask/metamask.jsx

```diff
@@ -173,9 +197,13 @@ export default function MetamaskProvider({ children }) {
 
   async function mint() {
     try {
-      const contentId = 'QmWoAd19aCmkax45YsaH1exYKHk5erv81fMRdMPxgGKmpz';
+      const contentId = "QmWoAd19aCmkax45YsaH1exYKHk5erv81fMRdMPxgGKmpz";
       const tokenId = `https://gateway.pinata.cloud/ipfs/${contentId}/samambaia01.png`;
```

Comment:
In the future makes more sense to store only the CID of the file

Reply:
I'm still confused how this is going to work

Reply:
I will explain better